config/routes.rb

```diff
@@ -3,4 +3,5 @@
   devise_for :users,
              controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
   root to: 'home#index'
+  resources :users
```

Comment:
only: [:index, :show]

Q - what do you plan to show on /users? and /users/:ID? what information?